pkgs/caddyserver/caddy/registry.yaml

```diff
@@ -0,0 +1,20 @@
+packages:
+  - type: github_release
+    repo_owner: caddyserver
+    repo_name: caddy
+    asset: caddy_{{trimV .Version}}_{{.OS}}_{{.Arch}}.{{.Format}}
+    format: tar.gz
+    description: Fast, multi-platform web server with automatic HTTPS
+    replacements:
+      darwin: mac
+    overrides:
+      - goos: windows
+        format: zip
+    checksum:
+      type: github_release
+      asset: caddy_{{trimV .Version}}_checksums.txt
+      file_format: regexp
+      algorithm: sha256
```

Comment:
This isn't sha256 but sha512. Let me fix.
https://github.com/caddyserver/caddy/releases/download/v2.5.2/caddy_2.5.2_checksums.txt

```
01af5ed08ec220fd3236d55083e863c78a961b496cb09eb216de5fad225e0226f936527cbf30efd4b4ac2071b8236925f819ed72435155c2b228d6604fce023a  caddy_2.5.2_freebsd_armv6.tar.gz
```

Reply:
a3c7fee689ba4fa8cf212285e7be9ea3f0e8ff40

Reply:
JFYI https://aquaproj.github.io/docs/reference/checksum